src/nodes/display/FilmNode.js

```diff
@@ -0,0 +1,60 @@
+import TempNode from '../core/TempNode.js';
+import { uv } from '../accessors/UVNode.js';
+import { luminance } from './ColorAdjustmentNode.js';
+import { addNodeElement, tslFn, nodeObject, vec3, vec4 } from '../shadernode/ShaderNode.js';
+import { uniform } from '../core/UniformNode.js';
+import { mix, fract, clamp, rand } from '../math/MathNode.js';
+import { timerLocal } from '../utils/TimerNode.js';
+
+class FilmNode extends TempNode {
+
+	constructor( textureNode, intensity = 0.5, grayscale = false ) {
+
+		super();
+
+		this.textureNode = textureNode;
```

Comment:
BTW: Should we make the name for this property more generic and rename it to `inputNode`? It might be a texture node but not always.

Reply:
I think textureNode would be one of the only exceptions where a node with the `TextureNode` class is necessary because of the samplers, so I think keeping the property name in this class would be better. `FilmNode` and other post-processing methods could be used for various effects even on materials, such as a television.

Reply:
Okay, let's keep `textureNode` for now and see how things shape up!

Reply:
Ah sorry, I thought this example had samplers, let me modify it then.